backend/functions/lib/interfaces/ICaseAssignmentRepository.d.ts

```diff
@@ -0,0 +1,13 @@
+import { ApplicationContext } from '../adapters/types/basic';
+import { CaseAttorneyAssignment } from '../adapters/types/case.attorney.assignment';
+
+export interface ICaseAssignmentRepository {
```

Comment:
modify the interface naming : append the word interface at the end. Remove the I at the beginning.

Reply:
Done.